src/vue.rs

```diff
@@ -177,6 +215,45 @@ impl zed::Extension for VueExtension {
         Ok(Some(initialization_options))
     }
 
+    fn language_server_additional_initialization_options(
+        &mut self,
+        _language_server_id: &zed::LanguageServerId,
+        target_language_server_id: &zed::LanguageServerId,
+        worktree: &zed::Worktree,
+    ) -> Result<Option<serde_json::Value>> {
+        match target_language_server_id.as_ref() {
+            "typescript-language-server" => Ok(Some(serde_json::json!({
+                "plugins": [{
+                    "name": "@vue/typescript-plugin",
+                    "location": self.get_ts_plugin_root_path(worktree)?.unwrap_or_else(|| worktree.root_path()),
+                }],
+            }))),
+            _ => Ok(None),
+        }
+    }
+
+    fn language_server_additional_workspace_configuration(
+        &mut self,
+        _language_server_id: &zed::LanguageServerId,
+        target_language_server_id: &zed::LanguageServerId,
+        worktree: &zed::Worktree,
+    ) -> Result<Option<serde_json::Value>> {
+        match target_language_server_id.as_ref() {
+            "vtsls" => Ok(Some(serde_json::json!({
+                "vtsls": {
+                    "tsserver": {
+                        "globalPlugins": [{
+                            "name": "@vue/typescript-plugin",
+                            "location": self.get_ts_plugin_root_path(worktree)?.unwrap_or_else(|| worktree.root_path()),
+                            "enableForWorkspaceTypeScriptVersions": true
```

Comment:
I'm currently using the hybrid mode. Then I need to add the languages for typescript plugin to be enabled for .vue sfc components. 

```suggestion
                            "enableForWorkspaceTypeScriptVersions": true,
                            "languages": ["typescript", "vue.js"],
```

Reply:
I think we should create a separate PR for it once this gets merged. It will be easier to pinpoint and debug in case it breaks anything else.